packages/portalnetwork/test/subprotocols/state/content.spec.ts

```diff
@@ -0,0 +1,117 @@
+import { describe, it, assert, assertType } from 'vitest'
+import testdata from './content.json'
+import {
+  AccountTrieProofType,
+  ContractStorageTrieProofType,
+  StateNetworkContentType,
+} from '../../../src/subprotocols/state/types.js'
+import { decodeStateNetworkContentKey } from '../../../src/subprotocols/state/util.js'
+import { fromHexString } from '@chainsafe/ssz'
+import { Account } from '@ethereumjs/util'
+import { Trie } from '@ethereumjs/trie'
+import { RLP } from '@ethereumjs/rlp'
+import { keccak256 } from 'ethereum-cryptography/keccak.js'
+
+describe('Account Trie Proof Content Type', async () => {
+  const contentKey = testdata.ATP.contentKey
+  const content = testdata.ATP.content
+
+  const decoded = decodeStateNetworkContentKey(fromHexString(contentKey)) as {
+    contentType: StateNetworkContentType.AccountTrieProof
+    address: Uint8Array
+    stateRoot: Uint8Array
+  }
+  const { witnesses, balance, nonce, codeHash, storageRoot } = AccountTrieProofType.deserialize(
+    fromHexString(content),
+  )
+  it('should deserialize content into account data and witnesses', async () => {
```

Comment:
In each of these tests where you decode a content key, can we spot check the actual value too, just for completeness?   So something like `assert.deepEquals(decoded.address, [the,actual,bytes,of,the,address])`